src/components/CreateVirtualFacilityModal.vue

```diff
@@ -170,6 +176,9 @@ export default defineComponent({
     markFacilityIdTouched() {
       (this as any).$refs.facilityId.$el.classList.add('ion-touched');
     },
+    isFacilityAlreadyExists() {
+      return this.virtualFacilities.some((facility: any) => facility.facilityId === this.formData.facilityId) || this.archivedFacilities.some((facility: any) => facility.facilityId === this.formData.facilityId)
```

Comment:
This will only check for facilityId in facilities those are displayed on the UI as we fetch facilities in batches, and will not check across all the facilities available.